stable/aad-pod-identity/Chart.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+description: Deploy components for aad-pod-identity
+name: aad-pod-identity
+version: 0.1.0
+appVersion: 1.4
```

Comment:
Is this connected in anyway to the product version - 1.5 ?

Reply:
Yes, changed to 1.5.